essential-projects/event_aggregator_contracts
# 💄 Split content of interfaces.ts

## What did you change?

- All interfaces are now located in separate files.

## How can others test the changes?

- Look through the source folder, you'll immediately see the changes.
- Build a solution that uses the event_aggregator_contracts project. The build should run exactly as before, since all changes made were exclusively cosmetic and non-functional. 

## PR-Checklist

Please check the boxes in this list after submitting your PR:
